remote/gogs/helper.go

```diff
@@ -70,13 +70,26 @@ func buildFromPush(hook *pushHook) *model.Build {
 		hook.Repo.URL,
 		fixMalformedAvatar(hook.Sender.Avatar),
 	)
+
+	var eventType string
+	var message string
+
+	switch {
+	case hook.RefType == "tag":
+		eventType = model.EventTag
+		message = "Tag " + hook.Ref
+	default:
+		eventType = model.EventPush
+		message = hook.Commits[0].Message
+	}
+
 	return &model.Build{
-		Event:     model.EventPush,
+		Event:     eventType,
 		Commit:    hook.After,
 		Ref:       hook.Ref,
 		Link:      hook.Compare,
 		Branch:    strings.TrimPrefix(hook.Ref, "refs/heads/"),
```

Comment:
we should change this to `"refs/tags/%s"`  when the hook type is tag. We can do this in a separate pull request, since I don't want to delay this merge any further :)